gluten-core/src/main/scala/io/glutenproject/extension/ColumnarOverrides.scala

```diff
@@ -188,10 +187,24 @@ case class TransformPreOverrides(
       )
       (projectExpressions, expressionPos)
     }
+    def getOutput(plan: SparkPlan) : Seq[Attribute] = {
```

Comment:
does this function suit for velox backend? @rui-mo please help to check

Reply:
I do not see any change of logic to Velox backend, so ti should be OK.